src/spirc.rs

```diff
@@ -172,6 +247,10 @@ impl SpircInternal {
                     self.is_active = false;
                     self.player.stop();
                 }
+
+                if frame.has_ident() && frame.has_goodbye() {
```

Comment:
I think this still is necessary. It stops the playing stream on librespot if you deselect librespot as target (by switching back to play on the controlling app). Now, librespot keeps playing, so two devices are playing at the same time (tested with Desktop OS X app).